components/cors-mgt/org.wso2.carbon.identity.cors.mgt.core/src/main/java/org/wso2/carbon/identity/cors/mgt/core/model/CORSApplication.java

```diff
@@ -0,0 +1,75 @@
+package org.wso2.carbon.identity.cors.mgt.core.model;
```

Comment:
Missing license

Reply:
Fixed in https://github.com/wso2/carbon-identity-framework/pull/3145